polaris-react/src/components/SkeletonDisplayText/SkeletonDisplayText.tsx

```diff
@@ -12,15 +16,25 @@ export interface SkeletonDisplayTextProps {
    * @default 'medium'
    */
   size?: Size;
+  /**
+   * Adjusts max-width
```

Comment:
Not sure why we're referring to it as text above but might be best to follow suit for now
```suggestion
   * Maximum width of the text
```